packages/mongodb-chatbot-ui/src/services/conversations.ts

```diff
@@ -134,10 +156,12 @@ export class ConversationService {
   }): Promise<MessageData> {
     const path = `/conversations/${conversationId}/messages`;
     const resp = await fetch(this.getUrl(path), {
+      ...this.fetchOptions,
       method: "POST",
       headers: {
         "Content-Type": "application/json",
         [CUSTOM_REQUEST_ORIGIN_HEADER]: getCustomRequestOrigin() ?? "",
+        ...this.fetchOptions.headers,
```

Comment:
For `createConversation` we spread `fetchOptions.headers` _under_ our `Content-Type` and custom request origin headers (i.e. we overwrite the custom headers with our own). Here in `addMessage` we spread _over_ our headers which is the opposite (i.e. we let users overwrite our headers with their own).

Should we be consistent about this across methods?